snapcraft/plugins/kbuild.py

```diff
@@ -166,13 +169,19 @@ def do_install(self):
                  ['CONFIG_PREFIX={}'.format(self.installdir)] +
                  self.make_install_targets)
 
-    def build(self):
+    def prep_build(self):
```

Comment:
we have a new keyword called prepare. This name would make it confusing. What about something like `prepare_config` or `configure`.